pkg/sql/catalog/descpb/structured.proto

```diff
@@ -1210,7 +1210,28 @@ message TableDescriptor {
   // gets incremented while preparing the table for ingestion.
   optional uint32 import_epoch = 59 [(gogoproto.nullable) = false, (gogoproto.customname) = "ImportEpoch"];
 
-  // Next ID: 60
+  // ImportType is the type of import that is running if
+  // ImportStartWallTime is set.
+  optional ImportType import_type = 60 [(gogoproto.nullable) = false, (gogoproto.customname) = "ImportType"];
```

Comment:
could this be a bool instead with the var name `epoch_based_import_runnning`? This feels cleaner then import_type==0  (i.e. `IMPORT_WITH_START_TIME_ONLY`)  implying no import running or non-epoch based import.

Reply:
I went back and forth on this. At the time it seemed likely that we might end up with a few more types if we eventually want to differentiate between empty-vs-non-empty as well like you had in your original PR. If so, I'd slightly prefer an Enum or a uint64 with different bits assigned to different properties of the import.

But, I can change it to a bool.

Reply:
fine by me!